features/user_can_compose_message.feature

```diff
@@ -0,0 +1,39 @@
+Feature: User can compose message
+    As a user
+    In order to communicate with others
+    I should be able to send messages
+
+    Background: 
+        Given the following user exists
+        | name | email          | password | password_confirmation |
+        | Greg | real@email.com | password | password              |
+        And I visit the site
+        When I click 'Login'
+        Then I should be on the login page
+        Then I fill in 'Email' field with 'real@email.com'
+        And I fill in 'Password' field with 'password'
+        And I click 'Log in'
+        Then I should see 'Signed in successfully.'
+        And I should be on welcome site
+        Then I click 'Inbox'
+        And I should be on the inbox site
+        And I click 'Compose'
+        And I should be on the compose message site
```

Comment:
The background has way too many steps and needs to be refactored

try `And is logged in`